nifti2/nifti2_io.c

```diff
@@ -564,7 +564,7 @@ void nifti_disp_lib_hist( int ver )
 *//*--------------------------------------------------------------------*/
 void nifti_disp_lib_version( void )
 {
-   printf("%s, compiled %s\n", gni_version, __DATE__);
```

Comment:
If date is useful for some reason here, it should be generated and passed in from a cmake variable that can be set and fixed.

Reply:
making reproducible builds is a good reason to omit `__DATE__`, thanks